src/Tests/newTask.test.js

```diff
@@ -1,7 +1,3 @@
-/**
- * @jest-environment jsdom
- */
-
 import List from '../app/modules/class.js';
```

Comment:
- [ ] You don't test any of the functions in your app, kindly create tests for the functions not just .pop() and .push(), these are just array methods. in this task, you need to test the functionality of the app to make sure they work properly

Reply:
Okay, I'll correct it. Thank you for your suggestion.